docker/clickhouse/user_defined_function.xml

```diff
@@ -288,4 +288,26 @@
         <command>aggregate_funnel_array_trends_test.py</command>
         <lifetime>600</lifetime>
     </function>
-</functions>
\ No newline at end of file
+
+    <function>
+        <type>executable_pool</type>
+        <name>get_survey_response</name>
+        <return_type>String</return_type>
+        <return_name>result</return_name>
+        <argument>
+            <type>String</type>
+            <name>properties</name>
+        </argument>
+        <argument>
+            <type>UInt64</type>
+            <name>question_index</name>
+        </argument>
+        <argument>
+            <type>String</type>
+            <name>question_id</name>
+        </argument>
+        <format>JSONEachRow</format>
+        <command>get_survey_response.py</command>
+        <lifetime>600</lifetime>
```

Comment:
whats this unit?

Reply:
for how long the function will leave before restarting the function process. from what I read it mostly helps with memory leaks. kept the same since it's the setting we have for other functions too